src/renderer/scss/component/_button.scss

```diff
@@ -1,224 +1,161 @@
 .btn {
-  min-width: var(--btn-height);
-  height: var(--btn-height);
-
-  align-items: center;
-  background-color: $lbry-teal-5;
-  border: none;
-  border-radius: var(--btn-radius);
-  color: $lbry-white;
-  cursor: pointer;
-  display: flex;
-  fill: currentColor; // for proper icon color
-  font-size: 12px;
-  justify-content: center;
-  padding: 10px;
+  fill: currentColor;
   position: relative;
-  text-decoration: none;
-  transition: all var(--animation-duration) var(--animation-style);
 
-  &:not(:disabled) {
+  .card__title & {
+    bottom: -0.5rem;
+    float: right;
+    font-size: 1rem;
+    position: relative;
   }
 
-  &:hover {
-    background-color: $lbry-teal-4;
+  &--alt {
   }
 
-  &:not(.btn--no-padding):not(.btn--link):not(.btn--no-style) {
-    .btn__content {
-      align-items: center;
-      display: flex;
-      padding: 0 8px;
-    }
+  &--constrict {
+    @include constrict;
   }
 
-  &.btn--alt {
-    &:not(:disabled) {
-      background-color: $lbry-white;
-      color: $lbry-black;
-    }
+  &--danger {
+  }
 
-    &:disabled {
-      background-color: transparent;
-      color: $lbry-gray-5;
-    }
+  &--disabled {
   }
 
-  &.btn--arrow {
-    color: $lbry-black;
-    width: var(--btn-arrow-width);
+  &--external-link,
+  &--link {
+    transition: color 0.2s;
+
+    &:not(:hover) {
+      color: $lbry-teal-5;
 
-    &:disabled {
-      opacity: 0.3;
+      html[data-theme='dark'] & {
+        color: $lbry-teal-3;
```

Comment:
After looking over this code more, I think we should move back to variables. I think it will be hard to keep consistency with this new pattern. (And add new theme options if we ever decided to do so).